modal/s3mount.py

```diff
@@ -0,0 +1,36 @@
+# Copyright Modal Labs 2022
+from dataclasses import dataclass
+from typing import List, Optional, Tuple
+
+from modal_proto import api_pb2
+from modal_utils.async_utils import synchronize_api
+
+from .secret import _Secret
+
+
+@dataclass
+class _S3Mount:
+    bucket_name: str
```

Comment:
This needs a [docstring like in _Volume](https://github.com/modal-labs/modal-client/blob/59e3bd97c5b09cce75b5ba6331ff1f4b79c9cbb3/modal/volume.py#L32).